cmd/build.go

```diff
@@ -34,6 +29,7 @@ var (
 	scmNew       = scm.New
 	osMkdirAll   = os.MkdirAll
 	useSudo      = false
+	loggerDone   chan interface{}
```

Comment:
Is it better to use chan struct{}?

Reply:
Thx. I fixed it.
I was under the mistaken impression that the size was too small even with `chan interface{}`.